mysqld_exporter.go

```diff
@@ -340,6 +348,139 @@ func (e *Exporter) scrape(ch chan<- prometheus.Metric) {
 			)
 		}
 	}
+
+	if *userStat {
+		informationSchemaUserStatisticsRows, err := db.Query("SELECT * FROM information_schema.USER_STATISTICS")
```

Comment:
I would prefer to have the columns listed instead of `*`.  This avoids ordering or feature changes between MySQL versions.